packages/celer-web-app/src/ui/components/DocLineComponent/DocLineBannerComponent.tsx

```diff
@@ -39,3 +44,29 @@ export const DocLineBannerComponent: React.FC<DocLineBannerProps> = ({docLine})=
 		</div>
 	);
 };
+
+export const NoteBannerComponent: React.FC<NoteBannerProps> = ({docLine, altNotesColor}) => {
+	const {notes, variables} = docLine;
+	const styles = useStyles();
+
+	const showTriangle = false;
```

Comment:
If this is always `false`, can we inline it?

Reply:
I can do that, but before I do, is there any situation where we might want a note banner to include the triangle? I can't think of a reason we would need that, but if you think that would be a nice option to have, I can add that capability.

Reply:
I think it's better with triangle since it would point to the previous line where the notes belong to